adapter-cqld4/src/main/resources/activities/baselines/cql-iot-dse.yaml

```diff
@@ -1,33 +1,29 @@
-# nb -v run driver=cql yaml=cql-iot tags=phase:schema host=dsehost
 description: An IOT workload with more optimal settings for DSE
-
 scenarios:
   default:
-    schema: run driver=cql tags==phase:schema threads==1 cycles==UNDEF
-    rampup: run driver=cql tags==phase:rampup cycles===TEMPLATE(rampup-cycles,10000000) threads=auto
-    main: run driver=cql tags==phase:main cycles===TEMPLATE(main-cycles,10000000) threads=auto
+    schema: run driver=cql tags==block:schema threads==1 cycles==UNDEF
+    rampup: run driver=cql tags==block:rampup  cycles===TEMPLATE(rampup-cycles,10000000) threads=auto
+    main: run driver=cql tags==block:"main-*.*" cycles===TEMPLATE(main-cycles,10000000) threads=auto
```

Comment:
Just an anecdotal note, the `main-*.*` pattern might confuse people, but I think it's fine as is. If I'm not mistaken, `main.*` is practically equivalent in that they would both match the same productions, given that `.*` will eat anything.

Reply:
That is correct, I agree we can adjust the `main-*` usage as needed.  One thing I did like is it helps indicate the naming uses that dash convention for the target block name.